src/client/components/Switch.tsx

```diff
@@ -23,31 +23,44 @@ const SwitchNobble = styled(Tick)`
   background-color: ${colorsV3.white};
 `
 
-interface SwitchProps {
-  value: boolean
-  onChange: (newValue: boolean) => void
-}
+/**
+ * Makes usage of a visually hidden checkbox for accessiblity reasons like
+ * to make it discoverable for assistence tools and to enable binding it with
+ * a <label> element.
+ * Based on https://css-tricks.com/inclusively-hidden/
+ */
+const InclusiveHiddenCheckbox = styled.input`
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+`
+
+type SwitchProps = Omit<React.InputHTMLAttributes<HTMLInputElement>, 'value'>
 
-export const Switch: React.FC<SwitchProps> = ({ value, onChange }) => (
-  <SwitchContainer
-    onTap={() => {
-      onChange(!value)
-    }}
-    animate={value ? 'checked' : 'unchecked'}
-    variants={{
-      checked: {
-        backgroundColor: colorsV3.gray900,
-        borderColor: colorsV3.gray900,
-      },
-      unchecked: {
-        backgroundColor: colorsV3.white,
-        borderColor: colorsV3.gray300,
-      },
-    }}
-    transition={{ duration: 0.1 }}
-    role="checkbox"
-    aria-checked={value}
-  >
-    <SwitchNobble />
-  </SwitchContainer>
+export const Switch = ({ className, ...checkboxProps }: SwitchProps) => (
+  <>
+    <InclusiveHiddenCheckbox type="checkbox" {...checkboxProps} />
+    <SwitchContainer
+      className={className}
```

Comment:
Why do we want to be able to pass `className`?

Reply:
TL;DR: To be able to control positioning/spacing outside `Switch`

I'm doing this because I wanted to be able to change its style in case I need to. Particularly in this case, I wanted to add some [spacing between](https://github.com/HedvigInsurance/web-onboarding/pull/743/files#diff-e5ba53806ba548c23022cb64eabc499c9f06d6cc1651ab055a8fea78717b6ec9R33) the checkbox (`Switch`) and the label. One might argue that I could solve the problem by adding the margin directly into `Switch` component but I think that by doing it in this way you end up with a more flexible component, leaving positioning/spacing matters to consumers, that usually have specifics about this, to handle them.

Reply:
Hm, I see. I generally think allowing for a `className` prop makes component a little _too_ flexible. Especially in this case where it's only for one purpose. How about a `gap` prop instead, or something similar?

Reply:
I guess this would be a great discussion for a design system 😉 

I personally really like the ability to `styled(Switch)` to adapt styles to the context. I worry more about too much flexibility when it comes to things like our Storyblok blocks.

Reply:
Yeah, a `gap` prop would work. But then I would need to make decisions about where the gap should be applied (top, right, bottom, left, or a mix) or if should I implemented using it `margin` or `padding`, etc. I guess it's just simpler to enable opt-in style customisation of all sort. But as @robinandeer mentioned, that's debatable.

Reply:
Realized that I maybe wasn't super clear. I was thinking that we'd probably never have _just_ a checkbox so a `gap` prop would rather be for the container - with both the checkbox and label - to set a space between them. There could also be a prop to decide whether or not the label comes before or after the checkbox. 

But yes, let's maybe not continue that discussion here 😄

Reply:
Oh, I see it now and I agree with you. I have came with the same conclusion while working on this. It's on my "personal backlog" open another PR with such refactoring and other stuff as well; e.g  rename `Switch` --> `Checkbox`.